pom.xml

```diff
@@ -65,6 +65,14 @@
 		</resources>
 		
 		<plugins>
+			<plugin>
```

Comment:
I'd suggest modifying this pom.xml reference as we did here: https://github.com/OHDSI/SkeletonCohortCharacterization/pull/4/files#diff-600376dffeb79835ede4a0b285078036R16 otherwise we'll get compiler warnings.

Reply:
I see this message when I try to build the pom. Is that caused by this @anthonysena ?

```
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for org.ohdsi:hydra:jar:0.0.4-SNAPSHOT
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-compiler-plugin is missing. @ line 68, column 12
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING] 
```